stable/kommander/charts/kommander-thanos/values.yaml

```diff
@@ -20,6 +20,8 @@ thanos:
     enabled: false
   bucket:
     enabled: false
+  rule:
```

Comment:
Just for my own edification what are we doing here?

Reply:
just being extra explicit about the rule component being disabled along with the others here